server/py/framework/routers/alerts.py

```diff
@@ -69,17 +70,25 @@ async def get_alert(
         Provide[framework.service.ServiceContainer.service]
     ),
 ) -> mlrun.common.schemas.AlertConfig:
+    client_version = request.headers.get("x-mlrun-client-version")
+    exclude_updated = client_version < "1.8.0" if client_version else False
```

Comment:
1. This should be in `alerts/main` - The routers are only responsible for routing the request to the handlers, and the handlers have the logic. Note that this file is part of the `framework`, so should not include business logic.
2. This is duplicated in `list_alerts` , can be a common function.